File: API/Entities/Product.cs
namespace API.Entities;

public class Product
{
    public int Id { get; set; }
    public required string Name { get; set; }
    public required string Description { get; set; }
    public long Price { get; set; }
    public required string ImageUrl { get; set; }
    public string? PublicId { get; set; }
    public required string Category { get; set; }
    public int Stock { get; set; }
    public char Size { get; set; }
    public required string Band { get; set; }
    public required string Genre{ get; set; }
    public int Sold { get; set; } = 0;


}

File: client/src/app/models/product.ts
export type Product = {
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  [x: string]: any;
  id: number;
  name: string;
  description: string;
  price: number;
  imageUrl: string;
  category: string;
  stock: number;
  size: string;
  band: string;
  genre: string;
};
